### PR TITLE
Merge skeleton-python-library

### PR DESCRIPTION
**Pull request by mcdonnnj**
# 

## 🗣 Description
This PR brings this project into the fold by merging [cisagov/skeleton-python-library](https://github.com/cisagov/skeleton-python-library). This includes migrating the project from Travis CI to GitHub Actions, 

## 💭 Motivation and Context
Since I will need to add some functionality to this project I thought it would be a good idea to bring it in line with other Python projects by merging in `skeleton-python-library`.

## 🧪 Testing
All automated tests pass with no issues. I tested the change to `defusedxml` manually and ensured that a JSON would still be generated correctly from sample XML.

## 🚥 Types of Changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (causes existing functionality to change)

## ✅ Checklist

- [x] My code follows the code style of this project.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

File: src/ade/assessment_data_export.py [hunk 1]
```diff
@@ -94,7 +96,7 @@ def export_jira_data(jira_base_url, jira_credentials_file, jira_filter, xml_file
     # Export XML data from Jira
     try:
         response = requests.get(
-            jira_url, auth=(jira_username, jira_password), verify=False
+            jira_url, auth=(jira_username, jira_password), verify=False  # nosec
```
Comment (jsf9k) [inline src/ade/assessment_data_export.py]: Why is nosec needed here?  It might be good to include a comment.

**Review by jsf9k — APPROVED** [review]
Approved, but see my one comment.

File: src/ade/assessment_data_export.py (hunk 1 above)
Comment (mcdonnnj) [reply src/ade/assessment_data_export.py]: Good catch, I mentioned it in the commit but forgot to add a code comment. Hopefully the comment in 2f19cc6 satisfies.

**Review by dav3r — APPROVED** [review]
Excelsior!  Fine work @mcdonnnj.  💪

**Pull request merged**